### PR TITLE
move apt stuff into one RUN

### PR DESCRIPTION
**Pull request by bebehei**
Hi @stucki 

I followed your work on the X5+ and seen your container here. Nice work. I'm currently trying to build lineageos with this container for myself.

I've updated the stack of apt-calls to better match [docker's best practices](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/#apt-get).

Ok, it's actually nothing in comparison to the later size lineageos root, but this also saves 50M (5%) in the container image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,23 +3,57 @@
 FROM ubuntu:16.04
 MAINTAINER Michael Stucki <michael@stucki.io>
 
-ENV DEBIAN_FRONTEND noninteractive
```
Comment (stucki) [inline Dockerfile]: Please kee this line and remove the `export` during `RUN` below.

File: Dockerfile [hunk 2]
```diff
@@ -3,23 +3,57 @@
...
-
-RUN sed -i 's/main$/main universe/' /etc/apt/sources.list
-RUN apt-get -qq update
-RUN apt-get -qqy upgrade
-
+RUN sed -i 's/main$/main universe/' /etc/apt/sources.list \
+ && export DEBIAN_FRONTEND=noninteractive \
```
Comment (stucki) [inline Dockerfile]: Remove this line

File: Dockerfile [hunk 3]
```diff
@@ -3,23 +3,57 @@
...
+      ccache \
+      file \
+      imagemagick \
+      nano \
+      rsync \
+      screen \
+      sudo \
+      tig \
+      wget \
+ && rm -rf /var/lib/apt/lists/*
```
Comment (stucki) [inline Dockerfile]: Rest looks good to me. Thank you!

**Review by stucki — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (bebehei) [inline Dockerfile]: Thanks for reviewing and merging so quick.

Setting `DEBIAN_FRONTEND` to noninteractive via `ENV` is discouraged. That's why I changed it. See [on this docker thread](https://github.com/moby/moby/issues/4032#issuecomment-34597177).

Therefore exporting `DEBIAN_FRONTEND` solely on shell during docker build process is the right choice.

Comment (stucki) [inline Dockerfile]: I see. That's new to me, but thanks for pointing out!

**Pull request merged**